src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/EnvelopeMapperTest.java

```diff
@@ -79,8 +82,19 @@ private void assertSameScannableItems(Envelope dbEnvelope, InputEnvelope zipEnve
 
         assertThat(dbEnvelope.getScannableItems())
             .extracting(this::convertToInputScannableItem)
-            .usingRecursiveFieldByFieldElementComparator()
+            .usingElementComparatorIgnoringFields("ocrData", "documentType")
             .containsAll(zipEnvelope.scannableItems);
+        assertThat(dbEnvelope.getScannableItems())
+            .extracting(ScannableItem::getOcrData)
+            .extracting(this::convertToInputOcrData)
+            .usingRecursiveFieldByFieldElementComparator()
+            .containsAll(
+                zipEnvelope.scannableItems.stream().map(item -> item.ocrData).collect(toList())
+            );
+        assertThat(dbEnvelope.getScannableItems())
+            .extracting(ScannableItem::getDocumentType)
+            .usingDefaultComparator()
+            .containsExactly(CHERISHED, OTHER, FORM, DocumentType.COVERSHEET);
```

Comment:
I think better to write another test just for scannableItems

Reply:
I agree with @mdayican. At first, we didn't do this conversion of document types, so there was just simple comparison to do. We should have added such test in the past when that bit got introduced. Having a separate test focusing on document type and subtype conversion would definitely be an improvement.

Reply:
@mdayican @lgonczar Added tests